### PR TITLE
#161348762 Users can like or dislike an Article

### PR DESCRIPTION
**Pull request by brucemakallan**
#### What does this PR do?
Adds feature for liking and disliking an Article

#### Description of Task to be completed?
- Display a specific article (based on the id from the URL)
- Add Like and Dislike icons for each article
- Collects the like and dislike counts for each article
- Updates like/dislike in case a user changes their mind about an article

#### How should this be manually tested?
- Go to the article's URL: `/articles/:articleId` e.g. `/articles/12`
- You should be able to see that articles full details
- You should be able to like / dislike that article

#### Any background context you want to provide?
You do not need to be logged in to view an article but You need to be logged in to like or dislike an article

#### What are the relevant pivotal tracker stories?
[#161348762](https://www.pivotaltracker.com/story/show/161348762)

#### Screenshots (if appropriate)
*Full article details page*
image

*Like, Dislike options on the article's page*
image

### CODE REVIEW

**Review by RachaelNantale — COMMENTED** [review]
why do we need this file?

**Review by dnuwa — DISMISSED** [review]

**Comment by brucemakallan** [comment]
That file is to avoid code repetition @RachaelNantale 
It creates a small section on Login and Signup pages for redirect links

**Review by RachaelNantale — DISMISSED** [review]
Looks good to me..

**Review by judeinno — DISMISSED** [review]

**Review by Sulaiman-Mozes — DISMISSED** [review]

**Comment by dnuwa** [comment]
@brucemakallan kindly look into fixing the Heroku review apps bug.
consider editing `"start": "serve -s dist"` in package.json to `"start": "serve dist"` and test it out.

File: src/actions/articleActions.js [hunk 1]
```diff
@@ -12,12 +12,50 @@ export const fetchArticlesFailure = errorMessage => ({
...
+      dispatch(getArticleAction(response.data.results));
+      return response;
+    })
+    .catch((error) => {
+      dispatch(showErrorAction({ ...alert, text: error.response ? error.response.data.results.error : 'Article not found' }));
+    });
+};
+
+export const getLikeStatusThunk = ({ articleId, token }) => (dispatch, getState) => {
+  const url = `https://ah-backend-thanos-staging.herokuapp.com/api/articles/${articleId}/like_status`;
```
Comment (tibetegya) [inline src/actions/articleActions.js]: replace the where ever there is the url with the APP_URL from the utils folder

Comment (brucemakallan) [reply src/actions/articleActions.js]: Alright

**Pull request merged**